package.json

```diff
@@ -46,7 +46,7 @@
     "jest-coverage": "yarn run jest -- --coverage",
     "lint": "eslint . --cache && yarn run lint-docs",
     "lint-docs": "eslint --config ./.eslintrc-docs.json --no-ignore --ext md ./docs/*.md",
-    "prettier": "prettier --write 'packages/*/src/**.js' --single-quote --trailing-comma=all --bracket-spacing=false",
+    "prettier": "prettier --write 'packages/*/src/**/*.js' --single-quote --trailing-comma=all --bracket-spacing=false",
```

Comment:
Does prettier automatically exclude nested `node_modules`? It has to, or this would take a long time to complete, right? If so, I can simplify some of my scripts here and there...